pycbc/events/stat.py

```diff
@@ -711,8 +711,8 @@ def coinc_lim_for_thresh(self, sngls_list, thresh, limifo,
         if not self.has_hist:
             self.get_hist()
 
-        lim_stat = [b['snglstat'] for a, b in sngls_list if a == limifo][0]
-        s1 = thresh ** 2. - lim_stat ** 2.
+        fixed_stat = [b['snglstat'] for a, b in sngls_list if a != limifo][0]
```

Comment:
Is this a change in logic (going from == to !=) or was this not working properly before?

Reply:
This wasn't working properly before - the statistic from the limifo was being used instead of the fixed IFO network

Looking again, I don't think this will actually work, so I will check it again